src/runner/index.js

```diff
@@ -123,16 +123,21 @@ export default class Runner extends EventEmitter {
     _validateRunOptions () {
         const concurrency = this.bootstrapper.concurrency;
         const speed       = this.opts.speed;
-        const proxyBypass = this.opts.proxyBypass;
+        let proxyBypass   = this.opts.proxyBypass;
 
         if (typeof speed !== 'number' || isNaN(speed) || speed < 0.01 || speed > 1)
             throw new GeneralError(MESSAGE.invalidSpeedValue);
 
         if (typeof concurrency !== 'number' || isNaN(concurrency) || concurrency < 1)
             throw new GeneralError(MESSAGE.invalidConcurrencyFactor);
 
-        if (proxyBypass)
-            assertType(is.string, null, '"proxyBypass" argument', proxyBypass);
+        if (proxyBypass) {
+            proxyBypass = Array.isArray(proxyBypass) ? proxyBypass : [ proxyBypass ];
```

Comment:
I think it's better to split proxy rules by comma here, instead of https://github.com/DevExpress/testcafe/pull/2313/files#diff-4602b4903a2087e7e79af24fa1e76b46R131